### PR TITLE
Feature/modal window/44 create modal component

### PR DESCRIPTION
**Pull request by lanniiee**
created modal window with styling
https://trello.com/c/FMIfvAs7/80-create-modal-wrapper-component

need to add this to transfer page for modal window to pop up and close :

import ModalWindow from "../../components/ModalWindow/ModalWindow";
import { useState } from "react";

const [modalVisible, setModalVisible] = useState(false);

const toggleModal = () => {
    setModalVisible(!modalVisible);
}

{modalVisible && <ModalWindow toggleModal={toggleModal} title="modal"> </ModalWindow>}

### CODE REVIEW

File: src/components/ModalWindow/ModalWindow.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.modal{
+    position: fixed;
+    width: 774px;
+    height: 735px;
+    left: 477px;
+    top: 191px;
+    padding: 20px;
+
+    background: #FFFFFF;
```
Comment (kirsty-nology) [inline src/components/ModalWindow/ModalWindow.scss:18]: should implement a variable here

File: src/components/ModalWindow/ModalWindow.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    background-color: hwb(0 0% 100% / 0.20);
+}
+
+.modal{
+    position: fixed;
+    width: 774px;
+    height: 735px;
+    left: 477px;
+    top: 191px;
+    padding: 20px;
```
Comment (kirsty-nology) [inline src/components/ModalWindow/ModalWindow.scss:16]: was there a reason pixels were used here?

File: src/components/ModalWindow/ModalWindow.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import "./ModalWindow.scss";
+import cross from "../../assets/images/modal-cross.png";
+
+const ModalWindow = ({title, toggleModal}) => {
+
+    return(
+        <div className="modal-background">
```
Comment (kirsty-nology) [inline src/components/ModalWindow/ModalWindow.jsx:7]: `className=modal`

File: src/components/ModalWindow/ModalWindow.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+            <div className="modal">
```
Comment (kirsty-nology) [inline src/components/ModalWindow/ModalWindow.jsx:8]: `className=modal__content`

File: src/components/ModalWindow/ModalWindow.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+                <h1>{title}</h1>
+                <img className="modal-cross" src={cross} alt="modal cross" onClick={toggleModal} />
```
Comment (kirsty-nology) [inline src/components/ModalWindow/ModalWindow.jsx:10]: `className=modal__image`

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

File: src/components/ModalWindow/ModalWindow.scss (hunk 2 above)
Comment (lanniiee) [reply src/components/ModalWindow/ModalWindow.scss]: this has now been changed to rem

File: src/components/ModalWindow/ModalWindow.jsx (hunk 4 above)
Comment (lanniiee) [reply src/components/ModalWindow/ModalWindow.jsx]: resolved

Comment (lanniiee) [reply src/components/ModalWindow/ModalWindow.jsx]: resolved

**Review by LewisLovett — CHANGES_REQUESTED** [review]
The modal window is not central on the screen
modal

**Review by kirsty-nology — APPROVED** [review]

**Review by LewisLovett — APPROVED** [review]

**Pull request merged**